src/styles/blocks/card.scss

```diff
@@ -0,0 +1,73 @@
+.card {
+  width: 200px;
+  border: 1px solid #f3f3f3;
+  padding: 32px 16px 16px;
+  display: flex;
+  flex-direction: column;
+
+  box-sizing: border-box;
```

Comment:
let's move this to `*` selector

Reply:
I don’t understand in which file to do this and I don’t understand why.

Reply:
Move this to the normalize file
Because you need this style in the whole of your page, with this - you will be easy implement everything according to your design

border-box tells the browser to account for any border and padding in the values you specify for an element's width and height. If you set an element's width to 100 pixels, that 100 pixels will include any border or padding you added, and the content box will shrink to absorb that extra width. This typically makes it much easier to size elements. box-sizing: border-box is the default styling that browsers use for the